src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToSubmitHandlerTest.java

```diff
@@ -653,6 +661,95 @@ public void givenIssueDirectionNoticeForAppealToProceedForPreValidCase_thenSetNo
+
+        callback.getCaseDetails().getCaseData().getAppeal().setBenefitType(BenefitType.builder().code(benefitType).build());
+
+        PreSubmitCallbackResponse<SscsCaseData> response = handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
+
+        assertNull(response.getData().getInterlocReviewState());
+        assertThat(response.getData().getDwpState(), is(DIRECTION_ACTION_REQUIRED.getId()));
+        assertNull(response.getData().getInterlocReferralReason());
+        assertThat(response.getData().getTimeExtensionRequested(), is("No"));
+    }
+
+    @Test
+    @Parameters({"pip, 35", "childSupport, 42"})
+    public void givenDirectionTypeOfAllowTimeExtensionWithPostValidState(String benefitType, int expectedResponseDays) {
+
+        callback.getCaseDetails().getCaseData().setState(VALID_APPEAL);
+        callback.getCaseDetails().getCaseData().setDirectionTypeDl(new DynamicList(DirectionType.GRANT_EXTENSION.toString()));
+        callback.getCaseDetails().getCaseData().setDirectionDueDate("11/07/2025");
+
+        callback.getCaseDetails().getCaseData().getAppeal().setBenefitType(BenefitType.builder().code(benefitType).build());
+
+
+        PreSubmitCallbackResponse<SscsCaseData> response = handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
+
+        assertThat(response.getData().getTimeExtensionRequested(), is("No"));
+        assertThat(response.getData().getDwpResponseDate(), is("11/07/2025"));
+        assertThat(response.getData().getDwpState(), is(DIRECTION_ACTION_REQUIRED.getId()));
+        assertNull(response.getData().getInterlocReferralReason());
+        assertNull(response.getData().getInterlocReviewState());
+
+
+    }
+
+    @Test
+    @Parameters({"pip, 35", "childSupport, 42"})
+    public void givenDirectionTypeOfExtensionRefusedWithListForHearing(String benefitType, int expectedResponseDays) {
+        callback.getCaseDetails().getCaseData().setDirectionTypeDl(new DynamicList(DirectionType.REFUSE_EXTENSION.toString()));
+        callback.getCaseDetails().getCaseData().setDirectionDueDate(null);
+        callback.getCaseDetails().getCaseData().setExtensionNextEventDl(new DynamicList(ExtensionNextEvent.SEND_TO_LISTING.toString()));
+        callback.getCaseDetails().getCaseData().getAppeal().setBenefitType(BenefitType.builder().code(benefitType).build());
+
+        PreSubmitCallbackResponse<SscsCaseData> response = handler.handle(ABOUT_TO_SUBMIT, callback, USER_AUTHORISATION);
+        assertThat(response.getData().getInterlocReviewState(), is(AWAITING_ADMIN_ACTION.getId()));
+        assertThat(response.getData().getState(), is(State.RESPONSE_RECEIVED));
+        assertThat(response.getData().getDwpState(), is(DIRECTION_ACTION_REQUIRED.getId()));
+        assertNull(response.getData().getInterlocReferralReason());
+        assertThat(response.getData().getTimeExtensionRequested(), is("No"));
+
+    }
+
+
```

Comment:
Can tidy up the space line between each test case, otherwise you may fail the format check, seems it only allow one space line between each test case.